### PR TITLE
Update linapple with dabone's changes

### PR DESCRIPTION
**Pull request by HerbFargus**
supports autolaunching disks (still needs a reset to start a game though) but this will remove the need for a start script for the apple 2.

should address #1055 and #1212

### CODE REVIEW

**Comment by joolswills** [comment]
This doesn't merge currently - your branch must have been out of date. I assume we will need to copy the default config file etc also still ?

**Comment by HerbFargus** [comment]
Ah yeh, my bad I'll sort that,  by copying the default config do you mean linapple.conf?

**Comment by joolswills** [comment]
Yeh - assuming it still uses that (so people can tweak it if needed)

**Comment by HerbFargus** [comment]
Ok it should be squashed and ready unless there are any other glaring issues with the module.

File: scriptmodules/emulators/linapple.sh [hunk 1]
```diff
@@ -37,31 +36,18 @@ function install_linapple() {
...
-        'splash.bmp'
         'Master.dsk'
         'README'
+        'README-linapple-pie'
+        'linapple.conf'
     )
     # install linapple.conf under another name as we will copy it
     cp -v "$md_build/linapple.conf" "$md_inst/linapple.conf.sample"
+    cp -vf "$md_build/Master.dsk" "$md_conf_root/apple2/Master.dsk"
 }
```
Comment (joolswills) [inline scriptmodules/emulators/linapple.sh]: This line shouldn't be needed, but also the install stage should only affect files in the $md_inst area.

Comment (joolswills) [inline scriptmodules/emulators/linapple.sh]: If the Master.dsk needs to exist in $md_conf_root/apple2 it needs to be copied at the configure stage as with the linapple.conf

Comment (HerbFargus) [inline scriptmodules/emulators/linapple.sh]: yeah it is needed for dabones changes but I can move it to the configure function

**Comment by HerbFargus** [comment]
when copying from the configure function instead of $md_build do I need to copy from $md_inst?

**Comment by joolswills** [comment]
Yeah - $md_inst needs to contain everything that is needed (apart from things that can be downloaded from configure such as wads for doom etc), so you install to $md_inst from install, then copy from there anything to the configuration folder as needed.

**Comment by joolswills** [comment]
that way when we archive up a binary package, it includes everything that is needed ($md_build would not exist in this case)

**Comment by HerbFargus** [comment]
make sense. I think it should be good now - I dont have my pi to test right now though

**Comment by joolswills** [comment]
I will give a a try in a bit also - Cheers!

**Comment by HerbFargus** [comment]
just as a reference its supposed to make it more friendly for frontend launching but as it was originally a computer there isnt really a clean way of running it all with a controller as it still requires a reset to start the disk and also other keyboard buttons to initialise a game, but at least this will remove the need for the start script

**Comment by HerbFargus** [comment]
I tested it and it seems to be working as intended.

**Comment by joolswills** [comment]
Apologies for being slow on this - still need to test etc.

**Comment by joolswills** [comment]
I'm sure it is fine actually.. I'll merge - don't want to be seen as holding back development ;-) thanks

**Pull request merged**